src/api/records.tsx

```diff
@@ -534,20 +583,29 @@ export const useThumbnails = (
 
 export const useRecordCount = (): UseQueryResult<number, AxiosError> => {
   const { apiUrl } = useAppSelector(selectUrls);
-  const { searchParams, filters } = useAppSelector(selectQueryParams);
+  const { searchParams, filters, functions } =
+    useAppSelector(selectQueryParams);
   const queryClient = useQueryClient();
   const projection = useAppSelector(selectSelectedIdsIgnoreOrder);
 
   return useQuery({
-    queryKey: ['recordCount', { searchParams, filters, projection }],
+    queryKey: ['recordCount', { searchParams, filters, projection, functions }],
```

Comment:
A question - the only reason we're adding functions here is because we need to account for function channels in the exists conditions correct? as otherwise functions don't affect the record count. Just wanted to make sure I understand :)

Reply:
Yes, that is the reason